### PR TITLE
2410: Make forecast and activity CSV upload look like actuals

### PR DESCRIPTION
**Pull request by dragon-dxw**
## Screenshots of UI changes

Forecasts: Before
image

Forecasts: After
image

-----

Activities: Before
image
Activities: After
image

## Next steps

- [x] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Remove unused translations

### CODE REVIEW

File: app/views/staff/reports/_activities_upload.html.haml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  = t("tabs.activities.upload.heading")
+
+= t("tabs.activities.upload.copy_html")
+
+.govuk-inset-text
+  = t("tabs.activities.upload.warning")
+
+.govuk-button-group
+  = link_to t("page_content.activities.button.download_template"), report_activity_upload_path(@report_presenter, format: :csv), class: "govuk-button govuk-button--secondary"
+  = link_to t("page_content.activities.button.upload"), new_report_activity_upload_path(@report_presenter), class: "govuk-button govuk-button"
```
Comment (CristinaRO) [inline app/views/staff/reports/_activities_upload.html.haml:17]: Is this an accidental duplication of `govuk-button`?

File: app/views/staff/reports/_forecasts_upload.html.haml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  = t("tabs.forecasts.upload.heading")
+
+= t("tabs.forecasts.upload.copy_html")
+
+.govuk-inset-text
+  = t("tabs.forecasts.upload.warning")
+
+.govuk-button-group
+  = link_to t("page_content.forecasts.button.download_template"), report_forecast_upload_path(@report_presenter, format: :csv), class: "govuk-button govuk-button--secondary"
+  = link_to t("page_content.forecasts.button.upload"), new_report_forecast_upload_path(@report_presenter), class: "govuk-button govuk-button"
```
Comment (CristinaRO) [inline app/views/staff/reports/_forecasts_upload.html.haml:17]: Same duplicate class.

File: app/views/staff/reports/activities.html.haml [hunk 3]
```diff
@@ -15,18 +15,7 @@
         .govuk-tabs__panel
           %h2.govuk-heading-l
             = t("tabs.report.activities.heading")
-
-          - if policy(@report_presenter).upload?
```
Comment (CristinaRO) [inline app/views/staff/reports/activities.html.haml:19]: I noticed that the conditional has disappeared in the move to the partial. Is this intentional?

File: config/locales/models/forecast.en.yml [hunk 4]
```diff
@@ -29,7 +29,7 @@ en:
         csv_file: Upload a spreadsheet containing forecast information in CSV format.
         csv_file_recover_from_error_html: Upload a spreadsheet containing forecast information in CSV format. We recommend <a href="%{link}" class="govuk-link">downloading this CSV template</a>.
         receiving_organisation: The organisation receiving the money from this transaction.
-        receiving_organisation_reference_html: For example, GB-COH-12345. To lookup codes or for more infomation see <a href="http://org-id.guide/" target="_blank" class="govuk-link">the organisation finder service (Opens in new window)</a>
+        receiving_organisation_reference_html: For example, GB-COH-12345. To lookup codes or for more information see <a href="http://org-id.guide/" target="_blank" class="govuk-link">the organisation finder service (Opens in new window)</a>
```
Comment (CristinaRO) [inline config/locales/models/forecast.en.yml:32]: Thank you for catching this typo!

I know it's out of scope, so it's optional to fix now, but the same typo appears in the `actual.en.yml` file, twice.

**Review by CristinaRO — COMMENTED** [review]
A couple of comments/questions!

File: config/locales/models/forecast.en.yml (hunk 4 above)
Comment (dragon-dxw) [reply config/locales/models/forecast.en.yml:32]: :+1:

File: app/views/staff/reports/_activities_upload.html.haml (hunk 1 above)
Comment (dragon-dxw) [reply app/views/staff/reports/_activities_upload.html.haml:17]: Good catch. It's a bug from where I've copy-pasted, possibly as a result of someone changing `govuk-button govuk-button--secondary` to no longer be secondary.

Submitting a separate commit to fix this bug throughout the codebase.

File: app/views/staff/reports/activities.html.haml (hunk 3 above)
Comment (dragon-dxw) [reply app/views/staff/reports/activities.html.haml:19]: Whoops, no. Fixed.

**Review by CristinaRO — APPROVED** [review]
Thank you, great work!

**Pull request merged**